pages/landing.tsx

```diff
@@ -0,0 +1,82 @@
+import styled from '@emotion/styled';
+import Header from '@src/components/common/Header';
+import { useRouter } from 'next/router';
+import React, { useState, useEffect } from 'react';
+import Image from 'next/image';
+import planetImage from '@assets/planet_main.svg';
+import IntroduceSentence from '@src/components/common/IntroduceSentence';
+
+function landing() {
+  const router = useRouter();
+  const messages: string[] = [
+    '나만의 우주를 만들어보세요.\n행성이 감정을 담아둘거예요',
+    '함께 느꼈던 감정을 행성에 기록할 친구들을\nYouniverse로 초대할 수 있어요.',
+  ];
+  const [message, setMessage] = useState(messages[0]);
+
+  useEffect(() => {
+    let timer = setInterval(() => {
+      switch (message) {
+        case messages[0]:
+          setMessage(messages[1]);
+          break;
+        case messages[1]:
+          router.push('/onboarding');
+          break;
+      }
+    }, 1500);
+    return () => clearTimeout(timer);
+  }, [message]);
+
+  return (
+    <>
+      <Header />
+      <Styled.Main>
+        <Styled.TextWrapper>
+          <IntroduceSentence />
+          <p>{message}</p>
+        </Styled.TextWrapper>
+        <Styled.PlanetWrapper>
+          <Image src={planetImage} alt="planet image" />
```

Comment:
alt는 이미지를 설명하는 부분이기 때문에, 
image / photo 와 같은 단어를 넣는 것은 무의미해요!
다른 lint 에서는 에러를 띄우기도 한답니다~

빼줘도 좋겠습니다!

Reply:
그리고 지금 PlanetWrapper에 `width: 243px`을 주었는데, image에 주는 것이 더 직관적이지 않을까요?

```suggestion
          <Image width="243px" src={planetImage} alt="planet image" />
```

Reply:
398f6e7742e59a4c7420f9b8106cab32b1f182f9